robotbase/src/main/java/frc/robot/state/RobotState.java

```diff
@@ -8,10 +8,13 @@ public class RobotState {
   public static enum State {
     INIT, // Robot is initializing
     DISABLED, // Robot is in disabled mode and no alliance is selected
+  }
+
+  public static enum IntakeState {
```

Comment:
Combine IntakeState and AmpState to show the travel of the note through the robot. This way we do not have to manage two states at the same time.